Drop-In/src/main/java/com/braintreepayments/api/VaultManagerFragment.java

```diff
@@ -0,0 +1,93 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ViewSwitcher;
+
+import androidx.annotation.VisibleForTesting;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.braintreepayments.api.dropin.R;
+
+import java.util.List;
+
+import static com.braintreepayments.api.DropInUIEventType.DISMISS_VAULT_MANAGER;
+
+public class VaultManagerFragment extends Fragment implements View.OnClickListener {
+
+    private ViewSwitcher loadingViewSwitcher;
+    private RecyclerView vaultManagerView;
+    private View baseView;
+
+    @VisibleForTesting
+    VaultManagerPaymentMethodsAdapter adapter;
+    @VisibleForTesting
+    DropInViewModel dropInViewModel;
+
+    public VaultManagerFragment() {
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
```

Comment:
Can we omit this `onCreate` override since we're only calling the super?

Reply:
Done 👍